changelogs/fragments/9012-dell-pwrbutton-requires-a-job-initiated-at-reboot.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - redfish_utils - Schedule a BIOS configuration job at next reboot when the
+    PwrButton config is changed
```

```suggestion
minor_changes:
  - redfish_utils module utils - schedule a BIOS configuration job at next reboot when the
    ``PwrButton`` config is changed (https://github.com/ansible-collections/community.general/pull/9012).
```
If we are to treat it like a bug fix, then it should be `bugfixes` instead of `minor_changes`, but I will defer that to people more familiar with these modules.

---

Thank you for your feedback. I've applied the suggestion while keeping in mind line length.